dcicutils/command_utils.py

```diff
@@ -0,0 +1,38 @@
+from .misc_utils import PRINT
+
+
+def yes_or_no(question, quick=None, default=None):
+    """
+    Loops asking a question interactively until it gets a 'yes' or 'no' response. Returns True or False accordingly.
+
+    :param question: The question to ask (without prompts for possible responses, which will be added automatically).
+    :param quick: Whether to allow short-form responses.
```

Comment:
I don't love how this interacts with `default`, seems confusing to me. Shouldn't both quick and default be booleans?

Reply:
I played with it interactively a bunch and it feel stupid to say
```
Do you want to do some horrible thing? (yes/no)
Spell out 'yes' for yes or 'no' for no, or just hit Enter to take the default without spelling anything out.
```

Reply:
I have a theory of how to fix this.

Reply:
OK, I've separated this into two functions, `yes_or_no` (which takes no arguments beyond the query string) and `y_or_n` (which takes only a default beyond the query string). The original function is renamed to internal-only, so the full generality of it is available but not promoted.

It still has a three-value default for `y_or_n` in order not to presuppose a True/False preference by default.
`>>> y_or_n("foo?")`
`foo? [y/n]:` ⏎
`Please answer 'y' or 'n'.`
`foo? [y/n]:` y⏎
`True`
`>>> y_or_n("foo?", default=True)`
`foo? [Y/n]:` ⏎
`True`
`>>> y_or_n("foo?", default=False)`
`foo? [y/N]:` ⏎
`False`
`>>> y_or_n("foo?", default=None)`
`foo? [y/n]:` ⏎
`Please answer 'y' or 'n'.`
`foo? [y/n]:`